Assets/HoloToolkit/Build/Editor/HoloToolkitCommands.cs

```diff
@@ -1,4 +1,9 @@
-﻿using HoloToolkit.Unity;
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I feel really strange about us adding MSFT copyright to files in public repos. Can MSFT really claim copyright over files that were produced by the public? In this case I wrote this on my free time, don't I own the copyright to it?